### PR TITLE
raidboss: p5s Raging Tail/Raging Claw Callout

### PR DESCRIPTION
**Pull request by Legends0**
Adds callout for when to move for Tail to Claw and Claw to Tail

### CODE REVIEW

File: ui/raidboss/data/06-ew/raid/p5s.ts [hunk 1]
```diff
@@ -109,13 +115,41 @@ const triggerSet: TriggerSet<Data> = {
...
       response: Responses.getFrontThenBack(),
     },
+    {
+      id: 'P5S Raging Tail Move',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '7A0C', source: 'Proto-Carbuncle', capture: false }),
+      infoText: (_data, _matches, output) => output.moveBehind!(),
+      outputStrings: {
+        moveBehind: {
+          en: 'Move (Behind)',
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/raid/p5s.ts:125]:
```suggestion
          en: 'Move Behind',
```

File: ui/raidboss/data/06-ew/raid/p5s.ts [hunk 2]
```diff
@@ -109,13 +115,41 @@ const triggerSet: TriggerSet<Data> = {
...
       netRegex: NetRegexes.startsUsing({ id: '770E', source: 'Proto-Carbuncle', capture: false }),
       durationSeconds: 5,
       response: Responses.getBackThenFront(),
     },
+    {
+      id: 'P5S Raging Claw Move',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '770F', source: 'Proto-Carbuncle', capture: false }),
+      condition: (data) => {
+        return ++data.clawCount === 6;
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/raid/p5s.ts:141]: For the sake of readability, can you put the increment and the equality on separate lines?

File: ui/raidboss/data/06-ew/raid/p5s.ts [hunk 3]
```diff
@@ -109,13 +115,41 @@ const triggerSet: TriggerSet<Data> = {
...
+      },
+      infoText: (_data, _matches, output) => output.moveFront!(),
+      run: (data) => {
+        data.clawCount = 0;
+      },
+      outputStrings: {
+        moveFront: {
+          en: 'Move (Front)',
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/raid/p5s.ts:149]:
```suggestion
          en: 'Move Front',
```

File: ui/raidboss/data/06-ew/raid/p5s.ts (hunk 2 above)
Comment (Legends0) [reply ui/raidboss/data/06-ew/raid/p5s.ts]: Done.

**Pull request merged**